lua/packer.lua

```diff
@@ -206,7 +309,7 @@ manage = function(plugin_data)
     return
   end
 
-  if plugins[name] and not plugins[name].from_requires then
+  if plugins[name] and not plugins[name].from_requires and not config.merge_duplicates then
```

Comment:
Maybe we should emit a `debug` level log message even if we are merging duplicates, in case of accidental override?

Reply:
Nevermind; I now see the log call on line 352.